supplementary_style_guide/introduction/about-guide.adoc

```diff
@@ -5,6 +5,8 @@
 
 This guide provides style guidelines and preferred term usage for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. It is intended to be used as a supplement to _The IBM Style Guide_, which is the primary source of guidance for Red Hat product documentation.
 
-This guide was created by the Red Hat Customer Content Services team, who is responsible for producing product documentation for Red Hat customers. It may also be used by upstream communities who want to align more closely with the standards used by Red Hat product documentation.
+The Red Hat Customer Content Services team created this guide as part of its responsibility for producing product documentation for Red Hat customers. Upstream communities who want to align more closely with the standards used by Red Hat product documentation may also use this guide. 
+
+= How to use this guide
```

Comment:
Typically there shouldn't be two level 1 headings in a file like this. I would make this a level 2 heading, or if you really think it needs to be a level 1, then move this section to another file.